src/api/OpenWith.js

```diff
@@ -48,13 +54,38 @@ class OpenWith extends Base {
             id: fileId,
             params,
             successCallback: openWithIntegrations => {
-                const formattedOpenWithData = this.formatOpenWithData(openWithIntegrations);
+                const consolidatedOpenWithIntegrations = this.consolidateBoxEditIntegrations(openWithIntegrations);
+                const formattedOpenWithData = this.formatOpenWithData(consolidatedOpenWithIntegrations);
                 successCallback(formattedOpenWithData);
             },
             errorCallback,
         });
     }
 
+    /**
+     * Removes the Box Edit SFC integration if the higher scoped Box Edit integration is present.
+     *
+     * @param {OpenWithAPI} openWithIntegrations - Open With integration items
+     * @return {OpenWithAPI} Open With Integrations with only one Box Edit integration
+     */
+    consolidateBoxEditIntegrations(openWithIntegrations: OpenWithAPI): OpenWithAPI {
+        let { items } = openWithIntegrations;
+        const boxEditIntegration = items.find(
+            integration => integration.app_integration.id === BOX_EDIT_INTEGRATION_ID,
+        );
+
+        if (boxEditIntegration) {
+            items = items.filter(integration => {
```

Comment:
This is mutating the `openWithIntegrations` param, since `items` is passed by reference, which isn't ideal. Can you just return the new object below, instead?